### PR TITLE
Incorporate res net lab

### PR DESCRIPTION
**Pull request by RuniteOre**
added links in further resources as per https://github.com/ipfs/ipfs-docs/issues/893
Just links in the footnotes, and one in the videos section of Concepts.

### CODE REVIEW

File: docs/concepts/ipns.md [hunk 1]
```diff
@@ -134,3 +134,6 @@ In the same way, you can republish a new version of your website under the same
 ## Alternatives to IPNS
 
 IPNS is not the only way to create mutable addresses on IPFS. You can also use [DNSLink](dnslink.md), which is currently much faster than IPNS and also uses human-readable names. Other community members are exploring ways to use blockchains to store common name records.
+
+## Further Resources
```
Comment (johnnymatthews) [inline docs/concepts/ipns.md:138]:
```suggestion
## Further Resources

```

File: docs/concepts/README.md [hunk 2]
```diff
@@ -24,6 +24,9 @@ Want the basics in theater mode? Check out these helpful videos for an overview
 
 - [IPFS: A Whiteboard Overview](https://www.youtube.com/watch?v=J-drqD2UebM)
 
+Check out ResNetLab on Tour for complete tutorials on IPFS and the Web 3.0 stack
```
Comment (johnnymatthews) [inline docs/concepts/README.md:27]:
```suggestion
Check out ResNetLab on Tour for complete tutorials on IPFS and the Web 3.0 stack:

```

**Review by johnnymatthews — APPROVED** [review]

**Comment by johnnymatthews** [comment]
Looks good! Thanks for this PR!

**Comment by RuniteOre** [comment]
Thanks!

**Pull request merged**